src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -981,33 +988,34 @@ It is also possible to change the ProActive Nodes Account using the 'Change Acco
 * The user can set a *memory limit* that will prevent the spawned processes to exceed a specified amount of RAM. If a spawned process or its child process requires more memory, it will be killed as well as its child processes. Note that this limit is disabled by default (0 means no limit) and a ProActive Node will require at least 128 MBytes.
 * It is possible to list all available *network interfaces* by clicking on the "Refresh" button and add the selected network interface name as a value of the `proactive.net.interface` property by clicking on "Use" button. See the ProActive documentation for further information.
-* The user can specify the *protocol* (PNP or PAMR) to be used by the ProActive Node for incoming communications.
+* The user can specify the *protocol* (PNP or PAMR) to be used by the ProActive Node for incoming communications. In case the selected protocol is PAMR, the user has to add `-Dproactive.pamr.router.address` JVM property to specify the router address which corresponds generally to the server hostname.
 * To ensure that a unique port is used by a ProActive Node, the initial port value will be incremented for each node process and given as value of the `-Dproactive.SELECTED_PROTOCOL.port` JVM property. If the port chosen for a node is already used, it is incremented until an available port number is found.
 
 Clicking on the 'Connection' tab, the window will look like this:
 
-image::config_editor_connection.png[]
+image::config_editor_connection.png[align=center, width=612, height=306]
 
 In the 'Connection' tab, the user can select between three types of connections:
 
 * *Local Registration* - creates a local ProActive node and registers (advertises) it in a local RMI registry. The node name is optional.
-* *Resource Manager Registration* - creates a local ProActive node and registers it in the specified Resource Manager. The mandatory Resource Manager's URL must be like *protocol://host:port/*. The node name and the node source name are optional. Since the Resource Manager requires authentication, the user specifies the file that contains the credential. If no file is specified the default one located in `%USERPROFILE%\.proactive\security` folder is used.
+* *Resource Manager Registration* - creates a local ProActive node and registers it in the specified Resource Manager. The mandatory Resource Manager's URL must be like *protocol://host:port/* (For PAMR protocol, no need to specify the host in case the router address is provided as a JVM property). The node name and the node source name are optional. Since the Resource Manager requires authentication, the user specifies the file that contains the credential. If no file is specified the default one located in `%USERPROFILE%\.proactive\security` folder is used.
```

Comment:
- the mandatory Resource Manager's URL must be like *protocol://host:port/* => the mandatory Resource Manager's URL must comply w/ the following format: 
- (For PAMR protocol, no need to specify the host in case the router address is provided as a JVM property) =>  In case of the PAMR protocol, the host is an integer assigned by the PAMR router. (0 for the router itself)

Reply:
In case of the PAMR protocol,, the URL is of the form pamr://resource_manager_id, which is by default pamr://0 (default value of the resource manager id). The PAMR router host and port are provided using JVM properties.